gophersnake.py

```diff
@@ -29,6 +29,7 @@
 import socket
 from threading import Thread
 import sys
+from io import BytesIO
```

Comment:
The io library is supported from Python 2.6 forward. This should be a solid choice for the vast majority of users at this point, given that most systems ship with at least 2.7.1

Reply:
It seems doubtful that 2.6 is still deployed in the wild, so targeting 2.7 is probably the right call.